tasks/frontEnd/src/components/ReviewForm.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <Form @submit="handleSubmit">
+    <TextField label="Shop name" v-model="review.shopName" autoComplete="off" required :error="shopNameError"></TextField>
+    <TextField label="Product name" v-model="review.productName" autoComplete="off" :error="productNameError"></TextField>
+    <TextField label="Your name" v-model="review.userName" autoComplete="off" :error="userNameError"></TextField>
+    <TextField type="email" label="Your email" v-model="review.userEmail" autoComplete="email" :error="userEmailError"></TextField>
+    <Text>Rating</Text>
+    <InlineStack gap="200">
+      <RadioButton v-for="n in 5" :label="n" :value="n" v-model="review.rating" :checked="review.rating == n" />
```

Comment:
Nit: Variable names like `n` can make it hard to debug, please try to use descriptive names where possible